www/js/services.js

```diff
@@ -3,7 +3,7 @@
 angular.module('emission.services', ['emission.plugin.logger',
                                      'emission.plugin.kvstore'])
 
-.service('CommHelper', function($rootScope) {
+.service('CommHelper', function($rootScope, Logger) {
```

Comment:
```suggestion
.service('CommHelper', function($rootScope) {
```

Logger is no longer needed here